homeassistant/components/mammotion/strings.json

```
@@ -0,0 +1,86 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
+      "already_in_progress": "[%key:common::config_flow::abort::already_in_progress%]",
+      "bluetooth_and_account_mismatch": "Bluetooth device not found in your account",
+      "missing_wifi_data": "Missing Wi-Fi configuration data",
+      "no_devices_found": "[%key:common::config_flow::abort::no_devices_found%]",
+      "no_devices_found_in_account": "No devices present in your account",
+      "no_longer_present": "Device is no longer present",
+      "not_supported": "Device not supported",
+      "reconfigure_successful": "[%key:common::config_flow::abort::reconfigure_successful%]"
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "flow_title": "Configure your Mammotion lawn mower",
+    "step": {
+      "bluetooth_confirm": {
+        "data": {
+          "stay_connected_bluetooth": "Keep Bluetooth connected"
+        },
+        "description": "Set up {name}"
+      },
+      "reconfigure": {
+        "data": {
+          "account_name": "Mammotion email or account number",
+          "password": "Mammotion account password",
+          "use_wifi": "Use Wi-Fi"
+        }
+      },
+      "user": {
+        "data": {
+          "address": "Device",
+          "stay_connected_bluetooth": "Keep Bluetooth connected"
+        },
+        "description": "Select your mower"
+      },
+      "wifi": {
+        "data": {
+          "account_name": "Mammotion email or account number",
+          "password": "Mammotion account password",
+          "use_wifi": "Use Wi-Fi (un-tick and submit to use Bluetooth)"
```

We should drop "submit" here, that's the obvious next step to save any changes, not just in that field.
Also "un-tick" is not a common wording in HA. Suggestion:

```suggestion
          "use_wifi": "Use Wi-Fi (deselect to use Bluetooth)"
```